### PR TITLE
Log the payload when creating a service instance

### PR DESCRIPTION
**Pull request by carbonin**
cc @gtanzillo this should give us what we're looking for to figure out if we're sending the right parameters when ordering a service

### CODE REVIEW

**Comment by carbonin** [comment]
@bdunne what do you think about making the logger a dev dependency here? I guess the assumption is that a `Rails.logger` will be available wherever this is used in production. Does that make sense?

**Comment by carbonin** [comment]
Also ping @eclarizio

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -14,6 +14,9 @@
 require "topological_inventory/core/ar_helper"
 TopologicalInventory::Core::ArHelper.load_environment!
 
+require 'manageiq/loggers'
+Rails.logger = ManageIQ::Loggers::Base.new(TopologicalInventory::Core::ArHelper.root.join("log/test.log"))
```
Comment (bdunne) [inline spec/spec_helper.rb]: Wouldn't it make more sense for it to be `TopologicalInventory::Core.root...`?

Comment (carbonin) [reply spec/spec_helper.rb]: Sure? I just left it where it was defined before. It was a private class method.

**Comment by bdunne** [comment]
> what do you think about making the logger a dev dependency here?

I don't think it's necessary especially since it's just for tests.

**Comment by carbonin** [comment]
>>what do you think about making the logger a dev dependency here?

>I don't think it's necessary especially since it's just for tests.

Okay, but that's what I did. So would you rather me stick it in the `Gemfile` rather than `gemspec`?

**Comment by bdunne** [comment]
>>>what do you think about making the logger a dev dependency here?

>> I don't think it's necessary especially since it's just for tests.

> Okay, but that's what I did. So would you rather me stick it in the Gemfile rather than gemspec?

Do we need `manageiq-loggers` at all?  I don't think so since it's only being used for tests.  I don't think we need the test logs formatted with out formatter.

**Comment by carbonin** [comment]
> Do we need manageiq-loggers at all? I don't think so since it's only being used for tests. I don't think we need the test logs formatted with out formatter.

Good point, I'll make it a regular logger.

**Comment by eclarizio** [comment]
Yeah, seems fine to me to have it be a regular logger since we don't care about the test logs anyway. 👍

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -14,6 +14,8 @@
 require "topological_inventory/core/ar_helper"
 TopologicalInventory::Core::ArHelper.load_environment!
 
+Rails.logger = Logger.new(TopologicalInventory::Core::ArHelper.root.join("log/test.log"))
```
Comment (bdunne) [inline spec/spec_helper.rb]: Is this necessary?

Comment (carbonin) [reply spec/spec_helper.rb]: Yes, that's the entire purpose of the PR ... The specs were failing before with no method `#info` for nil.

Comment (bdunne) [reply spec/spec_helper.rb]: Oh, I didn't know that

Comment (carbonin) [reply spec/spec_helper.rb]: Ah, sorry, `s/PR/commit/` 

Originally I just added the logging to the method, but then the specs started failing so I had to do this.

**Pull request merged**